benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -108,45 +133,34 @@ def _wait_for_delete_vm_scale(self, vm_num: str):
         """
         self._oc.wait_for_vm_delete(vm_name=f'{self._workload_name}-{self._trunc_uuid}-{vm_num}')
 
+    def _initialize_run(self):
+        """
+        Initialize run parameters
+        @return:
```

Comment:
This is another comment that either needs to be expanded on or removed.  It's actually not really accurate, since it doesn't just initialize parameters, it also initializes prometheus and more importantly has a side effect, creating the namespace.

Reply:
Updated, pls let me know if it more clear